README.md

```diff
@@ -66,4 +66,12 @@ Alternatively all steps can be reproduced using the following bash script:
 
 ```bash 
 bash bin/update_analysis.sh
-```
\ No newline at end of file
+```
+
+### Archived results
+
+Instead of rerunning the code from scratch archived results can be retrieved from the [OSF](https://osf.io/h6e8n/) which also contains an archive of the code used for the analysis.
+
+```r
```

Comment:
Maybe add a mention that this will overwrite any existing results files in `outputs/`?